django_celery_beat/schedulers.py

```diff
@@ -4,6 +4,7 @@
 import math
 from multiprocessing.util import Finalize
 
+import pytz
```

Comment:
Django v5__ drops support for `pytz` timezones.
* https://docs.djangoproject.com/en/5.0/releases/5.0/#features-removed-in-5-0